tests/integration/helpers/cluster.py

```diff
@@ -1369,7 +1389,11 @@ def create_dir(self, destroy_dir=True):
 
         print("Copy common configuration from helpers")
         # The file is named with 0_ prefix to be processed before other configuration overloads.
-        shutil.copy(p.join(HELPERS_DIR, '0_common_instance_config.xml'), self.config_d_dir)
+        if self.copy_users_xml:
+            shutil.copy(p.join(HELPERS_DIR, '0_common_instance_config.xml'), self.config_d_dir)
+        else:
+            shutil.copy(p.join(HELPERS_DIR, '0_common_instance_config.yaml'), self.config_d_dir)
```

Comment:
Let's call the variables to `config_xml_name` and `users_xml_name` (with default values `"config.xml"` and `"users.xml"`), this will allow us to write the copying code only once.